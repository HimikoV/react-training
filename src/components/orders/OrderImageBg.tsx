import { Grid, Box } from "@mui/material";

const OrderImageBg = () => {
  return (
    <Grid
      sx={{
        height: 315,
        width: "100%",
        display: "flex",
        alignItems: "center",
        justifyContent: "center",
        overflow: "hidden",
        flexWrap: "nowrap",
      }}
    >
      <Box
        component="img"
        sx={{
          height: 1934,
          width: 315,
          transform: "rotate(270deg)",
          objectPosition: "20% 0",
          objectFit: "cover",
        }}
        alt="BG for products Page"
        src="https://images.unsplash.com/photo-1603984719659-7af2c6f02a79?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
      />
    </Grid>
  );
};

export default OrderImageBg;
