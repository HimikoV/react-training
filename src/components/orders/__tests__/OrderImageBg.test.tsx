import { describe, it } from "@jest/globals";
import renderer from "react-test-renderer";
import OrderImageBg from "../OrderImageBg";

describe("testing <OrderImageBg/> render", () => {
  it("OrderImageBg Renders Correctly", () => {
    const orderBg = renderer.create(<OrderImageBg />).toJSON();
    expect(orderBg).toMatchSnapshot();
  });
});
