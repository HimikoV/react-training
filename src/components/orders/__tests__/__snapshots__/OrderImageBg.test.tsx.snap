// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing <OrderImageBg/> render OrderImageBg Renders Correctly 1`] = `
<div
  className="MuiGrid-root css-b7ixwv-MuiGrid-root"
>
  <img
    alt="BG for products Page"
    className="MuiBox-root css-oo92pk"
    src="https://images.unsplash.com/photo-1603984719659-7af2c6f02a79?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
  />
</div>
`;
